lib/cinematic.ts

```diff
@@ -495,17 +501,8 @@ class Cinematic {
             me.toggleFullScreen();
          });
 
-         document.addEventListener('fullscreenchange', () => {
-            if (this.isFullScreen()) {
-               this._container.dataset.fullscreen = true;
-               this._fullScreenButton.textContent = 'fullscreen_exit';
-               this._fullScreenButton.title = this.options.translations.exitFullscreen;
-            } else {
-               this._container.dataset.fullscreen = false;
-               this._fullScreenButton.textContent = 'fullscreen';
-               this._fullScreenButton.title = this.options.translations.fullscreen;
-            }
-         });
+         document.addEventListener('fullscreenchange', () => () => this.handleFullScreenChange());
```

Comment:
() => () => function()

sieht irgendwie nach einem `() =>` zu viel aus

Reply:
Guter Punkt, danke. Ist glaub auch das was Florian in der JS Datei angemerkt hat